content/en/functions/images/index.md

```diff
@@ -196,3 +198,9 @@ images.ImageConfig PATH
 favicon.ico: {{.Width}} x {{.Height}}
 {{ end }}
 ```
+
+{{% warning %}}
+If you try to call the function by passing it a resource or a resourceAdapter, you'll get an error. The argument must be a file path.
+{{%/ warning %}}
+
+Alternatively, you can access an image's parameters, like its `Width` and `Height`, by getting it as an image resource. For more information, see [The Image Page Resource](/content-management/image-processing/#the-image-page-resource).
```

Comment:
"If you wish to manipulate images, you can use [Hugo's image processing capabilities](/content-management/image-processing/#the-image-page-resource)."

No reason to advert image resources as an alternative for width and height parameters. This method here is much better, needs less resources and is probably cleaner for just this cause.

Reply:
In markdown render hooks, for example, imageConfig's difficult to use if you save images in page bundles, in your content/ directory.

Because .Destination will give you a path relative to the page. Bep recommended using mounts and resources for that purpose - something I couldn't figure out from the docs.

Hence the cross-link because to get an image's width and height, which you need to do lazy-loading without page shifts, you have two options: imageConfig and image resources.

It doesn't hurt to cross-reference related features a lot in docs, quite the contrary. This helps users find the thing they're trying to achieve, which may be the feature at hand or a similar yet different one, on a different page.

There's also a lot of missing info on the docs. For instance:

1.There's no list of the properties imageConfig gives you access to.
2. There's no mention of the function's speed and comparison with related features.

How about adding that information?

Reply:
In render hooks why not talking THERE about the two ways of obtaining info and their issues. 

As far as I know it's only height, width and color model that imageConfig does provide. It's the "bastard step brother" of page ressources. (https://gohugo.io/functions/images/#imageconfig). Not sure what the output of the color model is.

Reply:
Sure, I'll do this. I've removed the mention from this PR and will open another for that other contribution.